src/main/resources/ath-container/Dockerfile

```diff
@@ -13,15 +13,15 @@ RUN apt-get clean && \
         git \
         imagemagick \
         iptables \
-        firefox-esr \
+        firefox-esr=52.9.0esr-1~deb9u1 \
```

Comment:
FWIW I have had serious problems specifying exact versions in `apt` commands in `Dockerfile`s: after security fixes are released, vulnerable versions are _deleted_ from the repositories, and new `docker build`s start failing.

Reply:
@jglick  This is only needed until #457 gets merged, I am not very fond to do big changes in the Dockerfile until then... Just the minimum to get master working again.